styles.css

```diff
@@ -0,0 +1,100 @@
+@import url("https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #222;
+
+  font-family: "Rubik", sans-serif;
+
+  min-height: 100vh;
+  /* width: 100%; */
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  color: #fff;
+}
+
+main.container {
+  position: relative;
+  max-width: 100%;
+
+  margin: 1em;
+
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-template-rows: repeat(3, minmax(200px, auto));
+
+  text-align: center;
+
+  gap: 1em;
+
+  grid-auto-flow: dense;
+}
+
+.box {
+  background-color: #333;
+
+  padding: 1em;
+  display: grid;
+
+  place-items: center;
+
+  transition: 0.5s;
+}
+
+.box img {
+  max-width: 100px;
+}
+
+.box:hover {
+  background-color: #e91e63;
+}
+
+div.box:nth-child(1) {
+  grid-column: 1 / span 2;
+  grid-row: span 1;
+}
+.container .box:nth-child(2) {
+  grid-column: 3 / 4;
+  grid-row: 1 / 3;
+}
+.container .box:nth-child(3) {
+  grid-column: span 1;
+  grid-row: span 1;
+}
+.container .box:nth-child(4) {
+  grid-column: span 1;
+  grid-row: span 1;
+}
+.container .box:nth-child(5) {
+  grid-column: span 3;
+  grid-row: span 1;
+}
+.container .box:nth-child(6) {
+  grid-column: 2 / span 1;
+  grid-row: 2 / span 1;
+}
+.container .box:nth-child(7) {
+  grid-column: span 1;
+  grid-row: 2 / span 2;
+}
+
+@media screen and (max-width: 900px) {
+  main.container {
+    background-color: #e91e63;
+
+    grid-template-columns: repeat(autofill, minmax(50%, 1fr));
+    grid-template-rows: minmax(auto, auto);
+  }
+  .box {
+    grid-column: unset !important;
```

Comment:
unset means ?

Reply:
Well unset means to remove the old style applied. i.e. it switches from applied style to the default style. and here we have overriden it by using !important